markdown.md

```diff
@@ -0,0 +1,130 @@
+# Markdown Style Guide
+
+In general:
+
+- Be consistent with the file you found
+- Prioritize readability as plaintext over writability or rendered html
+
+
+## Line length
+
+Hard wrap lines at 80 characters whenever possible. An obvious exception is with
+hyperlinks. Move hyperlinks to their own line if they would exceed the 80 char
+limit.
+
+Rationale:
+
+
+## Links
+
+Prefer reference-style links. Always use reference-style links when the same
+URL is used in multiple files.
+
```

Comment:
Multiple places within the same file? If not, can you give an example of reference-style links when the URL is used in multiple files.